inputFiles/poromechanicsFractures/ExponentialDecayPermeability_EDFM_base.xml

```diff
@@ -0,0 +1,267 @@
+<?xml version="1.0" ?>
+
+<Problem>
+  <Solvers
+    gravityVector="{0.0, 0.0, 0.0}">
+    <SinglePhasePoromechanicsEmbeddedFractures
+      name="poroSolver"
+      targetRegions="{ Domain, Fracture }"      
+      initialDt="1e-3"
+      fracturesSolverName="fractureMechSolver"
+      flowSolverName="flowSolver"
+      solidSolverName="matrixMechSolver"      
+      logLevel="1">
+      <NonlinearSolverParameters
+        newtonTol="1.0e-5"
+        newtonMaxIter="10"
+        maxTimeStepCuts="1"/>
+      <LinearSolverParameters
+        directParallel="0"/>
+    </SinglePhasePoromechanicsEmbeddedFractures>
```

Comment:
We should add a test with the conforming fracture.

Reply:
@CusiniM A smoke test with conforming fracture is added and the running of integratedTests is checked on Pecan.